pamu2fcfg/pamu2fcfg.c

```diff
@@ -475,36 +475,36 @@ int main(int argc, char *argv[]) {
 
   r = fido_dev_info_manifest(devlist, DEVLIST_LEN, &ndevs);
   if (r != FIDO_OK) {
-    fprintf(stderr, "Unable to discover device(s), %s (%d)\n", fido_strerr(r),
-            r);
+    fprintf(stderr, "Unable to discover FIDO authenticator(s), %s (%d)\n",
+            fido_strerr(r), r);
     goto err;
   }
 
   if (ndevs == 0) {
     for (int i = 0; i < TIMEOUT; i += FREQUENCY) {
       fprintf(stderr,
-              "\rNo U2F device available, please insert one now, you "
+              "\rNo FIDO authenticator available, please insert one now, you "
               "have %2d seconds",
               TIMEOUT - i);
       fflush(stderr);
       sleep(FREQUENCY);
 
       r = fido_dev_info_manifest(devlist, DEVLIST_LEN, &ndevs);
       if (r != FIDO_OK) {
-        fprintf(stderr, "\nUnable to discover device(s), %s (%d)",
+        fprintf(stderr, "\nUnable to discover FIDO authenticator(s), %s (%d)\n",
                 fido_strerr(r), r);
         goto err;
       }
 
       if (ndevs != 0) {
-        fprintf(stderr, "\nDevice found!\n");
+        fprintf(stderr, "\nSecurity key found!\n");
         break;
       }
     }
   }
 
   if (ndevs == 0) {
-    fprintf(stderr, "\rNo device found. Aborting.                              "
+    fprintf(stderr, "\rNo FIDO authenticator found. Aborting.                  "
```

Comment:
Considering the use of `\r` here, should make sure this line is equally long to the "timer" line above. :slightly_smiling_face: